sdk/src/beta9/abstractions/endpoint.py

```diff
@@ -166,24 +252,27 @@ def __init__(
         name: Optional[str] = None,
         authorized: bool = True,
         autoscaler: Autoscaler = QueueDepthAutoscaler(),
+        task_policy: TaskPolicy = TaskPolicy(),
```

Comment:
do we want this here @jsun-m @dleviminzi ? Retries don't work for endpoints so might not be worth exposing

Reply:
whoops removed!

Reply:
👍